### PR TITLE
add side menu bar

### PR DESCRIPTION
**Pull request by shivams10**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+import Navbar from "./components/navbar";
+import "./App.css";
 
 function App() {
+  const [navVisible, showNavbar] = useState(false);
+
   return (
-    <div className="App">
-    </div>
+    <>
```
Comment (siddhi-rajput-22) [inline src/App.js]: Why have you wrapped it inside a fragment if you are using a single html tag

Comment (shivams10) [reply src/App.js]: Because In future i will add more components and routes in this fragment

Comment (siddhi-rajput-22) [reply src/App.js]: Then add this div when you require in future. Remove it fir now

Comment (shivams10) [reply src/App.js]: okay !

Comment (shivams10) [reply src/App.js]: issue has been fixed 
image

**Pull request merged**